### ISSUE TITLE
Uncaught TypeError: Cannot read property 'messages' of undefined

### ISSUE THREAD

**Issue by rstoenescu** [mozilla/html5-lint#21]
Got this error while processing 
``` js
<!DOCTYPE html><html><head>  <title>Title</title>  
 <!-- This is a comment --></head> <body>   

One</body>

 </html>   
```

``` js
var htmlLint = require('html5-lint');

htmlLint(file.contents.toString(), linterOpts, function(err, results) {
      _.forEach(results.messages, function(msg) {
```

**Comment by rstoenescu** [comment]
Ok, mystery solved... Seen issue #22 and it made me think a little then looked at the code and *yes* as the website was down html5-lint package couldn't process the input. Now that html5.validator.nu is up again, the code works and does not throw errors.
